src/arrayMethodSort.js

```diff
@@ -1,11 +1,27 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    let callBack = compareFunction;
+
+    if (!callBack) {
+      callBack = function (a, b) {
+        return String(a) > String(b) ? 1 : -1;
```

Comment:
When converting elements to strings, it's more robust to use `String(a).localeCompare(String(b))`. This will handle string comparisons more accurately for different locales.